### PR TITLE
[BUGFIX beta] Initializer/Instance Initializer Deprecations.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

**Comment by rwjblue** [comment]
Should pass properly once https://github.com/emberjs/data/pull/3665 lands and this is rebased (we need the fix from https://github.com/emberjs/data/commit/b6498c4f47e0031f5bfebc385bd29cb62667c947 for these tests).

**Comment by bmac** [comment]
I've merged https://github.com/emberjs/data/pull/3665. Do you mind rebasing this when you get a chance @rwjblue.

**Comment by rwjblue** [comment]
Rebased, should pass Travis for all three branches now.

File: packages/ember-data/lib/initializers/store.js [hunk 1]
```diff
@@ -22,7 +31,7 @@ export default function initializeStore(registry, application) {
   registry.register('serializer:-json-api', JSONAPISerializer);
 
 
-  if (!registry.has('service:store')) {
+  if (has(registry, 'service:store')) {
```
Comment (bmac) [inline packages/ember-data/lib/initializers/store.js]: Looks like this change if statement lost the `!`.

Comment (rwjblue) [inline packages/ember-data/lib/initializers/store.js]: Well, shoot.

**Comment by rwjblue** [comment]
@bmac - Thanks for helping me fix this up!

**Pull request merged**